pkg/azcli/embedded_configs.go

```diff
@@ -1,15 +1,19 @@
 package azcli
 
 var DefaultReadOnlyPatterns = `patterns:
```

Comment:
What's the relationship between DefaultReadOnlyPatterns  configs/readonly-operations.yaml? Should we maintain the same pattern in different places?